server.coffee

```diff
@@ -308,7 +321,8 @@ validate_context_property = (property_name) ->
         log.error "missing expected property #{property_name} on context"
         ctx.error = new Error("missing expected property #{property_name} on context")
         # req is not serializable
-        log.debug "context: #{JSON.stringify _.omit(ctx, "req")}"
+        log.error "context: #{JSON.stringify _.omit(ctx, "req")}"
+        log.error ctx['rendered_template']
```

Comment:
Why did you add this if it's already logging the contents of the context, and rendered_template is on the context?